README.md

```diff
@@ -129,6 +129,24 @@ $ yaegi cmd/yaegi/yaegi.go
 >
 ```
 
+Or for Go scripting in the shebang line:
+
+```console
+$ cat /tmp/test
+#!/usr/local/bin/yaegi
```

Comment:
`#!/usr/bin/env yaegi` is better (no hardcoding of yaegi location)

Reply:
done, sorry for the delay :)